app/views/docs/configuration.phtml

```diff
@@ -0,0 +1,65 @@
+
+<h2><a href="#delivery" id="delivery">Setup Email and SMS Delivery</a></h2>
+<p>
+    Appwrite requires an SMTP service to deliver emails. Email delivery is required for features like password recovery, account verification, MagicURL authentication, and team invites. You can use a third-party SMTP service or host your own SMTP server.
+</p>
+<p><a href="/docs/email-delivery">Set up email delivery</a></p>
+
+<p>
+    Appwrite requires a third-party SMS delivery service to send SMS messages. SMS delivery is required for phone authentication.
+</p>
+<p><a href="/docs/sms-delivery">Set up SMS delivery</a></p>
+
+<h2><a href="#functions" id="functions">Configure Function Runtimes</a></h2>
+<p>
+    Not all function runtimes are enabled by default. Enable the runtimes that you need and disable unused runtimes to save disk space on your server.
+    To enable a runtime, add it to the <code>_APP_FUNCTIONS_RUNTIMES</code> environment variable as a comma separated list. 
+</p>
+
+<p>
+    The example below would enable Dart 2.15, .Net 6.0, and Java 18 runtimes.
+</p>
+
+<div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
+    <pre class="line-numbers"><code class="prism language-bash" data-prism>_APP_FUNCTIONS_RUNTIMES=dart-2.15, dotnet-6.0, java-18.0</code></pre>
+</div>
+
+You can find a full list of supported runtimes on the <a href="/docs/environment-variables#functions">environment variables</a> page.
+
+<p>You can also configure the maximum timeout that can be set on individual Appwrite functions. The maximum configurable timeout can be increased by changing the <code>_APP_FUNCTIONS_TIMEOUT</code> environment variable. This environment variable changes the configurable maximum but does not alter existing individual configurations.</p>
+
+<h2><a href="#storage" id="storage">Storage Adaptors</a></h2>
+<p>Appwrite's Storage Service can be configured to store files locally, or with self-hosted and cloud storage services. By default, Appwrite's Storage Service <b>stores files on your server's local storage</b>. If you expect large volumes of data or the need to have scalable data storage, you may choose to use a separate storage service.</p>
+
+<p>Appwrite supports AWS S3, Digital Ocean Spaces, Backblaze, Linode, and Wasabi as storage adaptors. Some of these services can be self-hosted, just like Appwrite.</p>
+
+<p>You can select which storage adaptor to use by setting the <code>_APP_STORAGE_DEVICE</code> environment variable. Valid values are <code>local</code>, <code>s3</code>, <code></code>, <code>backblaze</code>, <code>linode</code> and <code></code>. Each storage adaptor requires its own set of environment variables to configure.</p>
+
+<p>The maximum file size allowed for file upload is controlled by the <code>_APP_STORAGE_LIMIT</code> environment variable, which defaults to 30 MB. See <a href="/docs/environment-variables#storage">Environment Variables</a> for more information.</p>
```

Comment:
Can we be more clear and mention this is "per file" size limit ? and not to the entirety of the storage service?

Reply:
Done